src/app/window-handler.ts

```diff
@@ -160,12 +164,28 @@ export class WindowHandler {
      */
     public async createApplication() {
 
-        this.updateVersionInfo();
+        await this.updateVersionInfo();
         this.spellchecker = new SpellChecker();
         logger.info(`window-handler: initialized spellchecker module with locale ${this.spellchecker.locale}`);
 
         logger.info('window-handler: createApplication mainWinPos: ' + JSON.stringify(this.config.mainWinPos));
 
+        let {isFullScreen, isMaximized} = this.config.mainWinPos ? this.config.mainWinPos : {isFullScreen: false, isMaximized: false};
+
+        this.url = WindowHandler.getValidUrl(this.userConfig.url ? this.userConfig.url : this.globalConfig.url);
+        logger.info(`window-handler: setting url ${this.url} from config file!`);
+
+        if (this.globalConfig.url.startsWith('https://my.symphony.com') && !this.userConfig.url) {
```

Comment:
Will this not show the dialog every time for a user that want to use my.symphony.com

Don't you need a flag e.g. showWelcome

Reply:
No @johankwarnmarksymphony. It shows the dialog only if the url is not set in user config file.

And, once a user sets the url, we update that in the user config file, so, this condition won't come into picture after that.

```
ipcMain.on('set-pod-url', async (_event, newPodUrl: string) => {
             await config.updateUserConfig({url: newPodUrl, mainWinPos: { ...this.mainWindow!.getPosition(), ...{ height: 900, width: 900 } } });
             app.relaunch();
             app.exit();
         });
```